### DISCUSSION TITLE
Are these loading graphics normal?

### DISCUSSION THREAD

**Discussion by Null-B** [community/community#20698] [Codespaces]
![image](https://user-images.githubusercontent.com/76712316/179374765-88160537-fb75-4577-b18f-5eef1da0277d.png)
Hi i wanted to ask dose anyone else get these messed up loading graphics, because i tried it on a another laptop and it looked different.
is something broken with my codespaces? or is this just a feature?

**Comment by Fengyi-Li** [comment]
This is not expected. Can you connect to the codespace you created successfully? Does this issue reproduce consistently on your side?
